neo4jlabs-plugins.json

```diff
@@ -22,6 +22,12 @@
       "dbms.security.procedures.unrestricted":"algo.*"
     }
   },
+  "n10s": {
+    "versions": "https://raw.githubusercontent.com/neo4j-labs/neosemantics/master/versions.json",
```

Comment:
Could we change this to use a GitHub pages url - that allows for better tracking and a level of indirection that could come in handy.
e.g. `https://neo4j-labs.github.io/neosemantics/versions.json`

Reply:
done!